pyserini/search/pysearch.py

```diff
@@ -292,3 +293,25 @@ def get_topics(collection_name):
         for key in topics.get(topic).keySet().toArray():
             t[topic][key] = topics.get(topic).get(key)
     return t
+
+class SimpleNearestNeighborSearcher:
+
+    def __init__(self, index_dir: str):
+        self.object = JSimpleNearestNeighborSearcher(JString(index_dir))
+
+    def search(self, q: str, k=10) -> List[List[JSimpleNearestNeighborSearcherResult]]:
```

Comment:
Is the return type here correct?

It's dispatch to this method right? `public Result[] search(String id, int d)`
https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/search/SimpleNearestNeighborSearcher.java#L78

Reply:
yes, correct. I was fixing it :)